doc/source/contribution/documentation.md

```diff
@@ -49,6 +45,19 @@ Clone Mroonga repository to working directory. Don't forget to do "Git configura
 % git remote add upstream git@github.com:mroonga/mroonga
 ```
 
+### Required softwares
+
+Generating Mroonga documentations needs [Sphinx](https://www.sphinx-doc.org/) and MySQL's source and build directories.
```

Comment:
We also need `mysql_config` and installing groonga for generating documents.

Reply:
fix: [2715e01](https://github.com/mroonga/mroonga/pull/723/commits/2715e01e5ac3828bf485f7b7e0dd6921e071987c) Thanks, I added the required softwares.